primitives/runtime/src/lib.rs

```diff
@@ -619,18 +620,25 @@ pub enum TokenError {
 	Frozen,
 	/// Operation is not supported by the asset.
 	Unsupported,
+	/// Account cannot be created for a held balance.
+	CannotCreateHold,
+	/// Withdrawal would cause unwanted loss of account.
+	UnwantedAccountRemoval,
```

Comment:
This reads like it's removing an unwanted account, while the meaning here is supposed to be an unwanted removal of an account. How about we reword the variant to the following?

```suggestion
	UnwantedRemovalOfAccount,
```

Or switch `Removal` with `Loss`?